src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -170,7 +170,7 @@ private void CheckTrainingParameters(ImageClassificationEstimator.Options option
             Host.CheckNonWhiteSpace(options.LabelColumn, nameof(options.LabelColumn));
             Host.CheckNonWhiteSpace(options.TensorFlowLabel, nameof(options.TensorFlowLabel));
 
-            if (_session.graph.OperationByName(options.TensorFlowLabel) == null)
+            if (_session.graph.OperationByName(_labelTensor.name.Split(':')[0]) == null)
```

Comment:
>_labelTensor.name.Split(':')[0] [](start = 47, length = 31)

If this is the name of the operation you are searching for in the graph, then the error message should mention the correct name. In this case, the options argument is not needed here.

Reply:
the options argument is not used anymore. The _labelTensor has the correct name that the error message would need

---
In reply to: [324450724](https://github.com/dotnet/machinelearning/pull/4205#discussion_r324450724) [](ancestors = 324450724)